spec/mailers/users_mailer_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe UsersMailer, type: :mailer do
+  before(:each) do
+    ActionMailer::Base.delivery_method = :test
```

Comment:
remove these 3 lines
TEST NEVER SENDING EMAIL! NEVER
in test.rb leave default configuration